saleor/graphql/csv/types.py

```diff
@@ -63,12 +74,21 @@ def resolve_url(root: models.ExportFile, info):
         return info.context.build_absolute_uri(content_file.url)
 
     @staticmethod
-    def resolve_created_by(root: models.ExportFile, info):
+    def resolve_user(root: models.ExportFile, info):
         requestor = get_user_or_app_from_context(info.context)
-        if requestor == root.created_by or requestor.has_perm(
-            AccountPermissions.MANAGE_USERS
+        if (
+            requestor == root.user
+            or requestor.has_perm(AccountPermissions.MANAGE_USERS)
```

Comment:
I think it should be only `MANAGE_STAFF` permission. 
Export could be created only by staff users.